packages/zsh/build.sh

```diff
@@ -28,6 +31,23 @@ TERMUX_PKG_RM_AFTER_INSTALL+="
 share/zsh/${TERMUX_PKG_VERSION}/functions/_pkg5
 "
 
+termux_step_pre_configure() {
+	## large file fix
+	if test "$TERMUX_ARCH" = arm; then
```

Comment:
Probably needed on i686 as well

Reply:
I dont own an i686 device (unable to test, should I also add i686 in commit so that the CI could test this?)

Reply:
> I dont own an i686 device

That's not surprising as almost none ever existed. 😄 Probably nobody ever used i686, so we just add support if it compiles, no need to test it.

Reply:
>Since 2012 also Android™ devices with Intel x86 Atom CPUs are available. Some examples of Android™ devices running on Intel Architecture are: Samsung Galaxy Tab 3 10.1, Asus MemoPad FHD 10, Dell Venue 7/8, Motorola Razr I, Lenovo K900 or more recently the HP 7.

https://murlengine.com/news/?murllang=en&murlnews=2014-08-12_android_x86_support

@buttaface I guess it's now your responsibility to find one of those devices still alive, port android 5 and 7 on them and test. 😉

Reply:
> @buttaface I guess it's now your responsibility to find one of those devices still alive, port android 5 and 7 on them and test. 😉

qemu+android x86, although it's slow 🙄 .

Reply:
i have reworked the commit, now the commit also targeting 64-bit arch.

Reply:
>qemu+android x86, although it's slow roll_eyes .

One can create x86 avd too till like android 11 and won't be slow depending on host device. But a "real device" though...